src/gql/generated/types.ts

```diff
@@ -0,0 +1,622 @@
+export type Maybe<T> = T | null;
+/** All built-in and custom scalars, mapped to their actual values */
+export type Scalars = {
+  ID: string;
+  String: string;
+  Boolean: boolean;
+  Int: number;
+  Float: number;
+  Time: any;
+  Duration: any;
+  StringMap: any;
```

Comment:
scalars can be mapped to their actual values by mapping them to their respective types in the config file https://graphql-code-generator.com/docs/plugins/typescript#configuration